### PR TITLE
Plugin to add a task to Wunderlist

### PR DESCRIPTION
**Pull request by AlAdler**

### CODE REVIEW

**Comment by nate-parrott** [comment]
sweet! this is something people have been asking for.

File: PluginDirectories/1/wunderlist.bundle/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "categories" : [
+    "Utilities"
+  ],
+  "displayName" : "Task to Wunderlist",
+  "name" : "Wunderlist",
```
Comment (nate-parrott) [inline PluginDirectories/1/wunderlist.bundle/info.json]: for future reference, the `name` value should be the same as the `.bundle` file's name (minus the extension), including capitalization. I'll fix it

**Comment by xremix** [comment]
@AlAdler oooohh awesome! Already searched how to develop against wunderlist but couldn't figure that out!

File: PluginDirectories/1/wunderlist.bundle/info.json (hunk 1 above)
Comment (AlAdler) [inline PluginDirectories/1/wunderlist.bundle/info.json]: OK, I'll remember that for the next plugin.

**Comment by AlAdler** [comment]
What other integrations have people been asking for?

On Thu, Jan 15, 2015 at 9:29 PM, Nate Parrott <notifications@github.com>
wrote:

> sweet! this is something people have been asking for.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nate-parrott/Flashlight/pull/272#issuecomment-70145945>
> .
>



-- 

Alain Adler

**Comment by nate-parrott** [comment]
Here's some: http://ideaboardapp.appspot.com/flashlight-plugin-ideas

**Comment by AlAdler** [comment]
Hi @nate-parrott, you should remove the Wunderlist plugin from the lists, it's not working, Wunderlist have closed their API.

**Comment by jonathanj** [comment]
Wunderlist [recently announced](https://developer.wunderlist.com/) some new `API`s, it would be great to get the Wunderlist plugin working again!

**Pull request merged**